### ISSUE TITLE
El delete no funciona en index.py

### ISSUE THREAD

**Issue by akevalion** [jpiro80/Proyecto1#4]
delete deberia eliminar un producto

**Comment by jpiro80** [comment]
Sí, y tampoco está funcionando el de editar. Ahí subí lo más actualizados que pude los dos archivos para revisar.
